src/detail/sqlite_backend.cc

```diff
@@ -49,11 +50,71 @@ expected<data> from_blob(const void* buf, size_t size) {
     return {ec::invalid_data};
 }
 
+// Find name in options and verify it starts with the given prefix,
+// if the prefix-stripped part of the value is found in allowed,
+// set result to the value.
+bool extract_optional_enum_option(
+  const broker::backend_options& options, const std::string& name,
+  const std::string_view& prefix,
```

Comment:
A string view is a pointer and a size. Best practice is to pass this by value and give the compiler more room for optimization by not forcing pointer indirections.

```suggestion
  std::string_view prefix,
```